packages/shared/src/components/BlockItemWrapper/BlockItemWrapper.tsx

```diff
@@ -14,21 +15,29 @@ export const BlockItemWrapper = ({
     isDragging,
 }: PropsWithChildren<BlockItemWrapperProps>) => {
     const [isFlyoutOpen, setIsFlyoutOpen] = useState(false);
+    const [isFocused, setIsFocused] = useState(false);
+    const { focusWithinProps } = useFocusWithin({
+        onFocusWithin: () => setIsFocused(true),
+        onBlurWithin: () => setIsFocused(false),
+    });
+
     return (
         <div
+            {...focusWithinProps}
```

Comment:
Can this library potentially be substituted with simple tailwind keywords, like `group, focus, focus-within` ?

e.g. add `group` to parent and then in the children you can do `group-focus:opacity:0`